Lesson30/Firat/my-app/src/components/Exercise8/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { useState } from "react";
+
+export const Exercise8 = () => {
+  const [list, setList] = useState(["Item 1", "Item 2", "Item 3", "Item 4"]);
+
+  const handleRemove = (index) => {
+    const newList = [...list];
+    newList.splice(index, 1);
+    setList(newList);
+  };
+
+  return (
+    <div>
+      <ul>
+        {list.map((item, index) => (
+          <li key={index}>
+            {item}
+            <button onClick={handleRemove}>remove</button>
```

Comment:
You forgot to pass the index parameter to the handleRemove function so right now a random list item is being removed.
```suggestion
            <button onClick={() => handleRemove(index)}>remove</button>
```